apps/settings/js/panels/about_more_info/hardware_info.js

```diff
@@ -85,6 +87,11 @@ define(function(require) {
             this._refreshBluetoothAddress.bind(this));
           this._refreshBluetoothAddress(Bluetooth.address);
         }
+        else {
```

Comment:
Bluetooth module is loaded from `modules/bluetooth/bluetooth_context` so the else part is unnecessary